vmm/src/config.rs

```diff
@@ -2086,12 +2111,73 @@ impl RestoreConfig {
             .map_err(Error::ParseRestore)?
             .unwrap_or(Toggle(false))
             .0;
+        let net_ids = parser
+            .convert::<StringList>("net_ids")
+            .map_err(Error::ParseRestore)?
+            .map(|v| v.0);
+        let net_fds = parser
+            .convert::<IntegerList>("net_fds")
+            .map_err(Error::ParseRestore)?
+            .map(|v| v.0.iter().map(|e| *e as i32).collect());
 
         Ok(RestoreConfig {
             source_url,
             prefault,
+            net_ids,
+            net_fds,
         })
     }
+
+    pub fn validate(&self, vm_config: &VmConfig) -> ValidationResult<()> {
+        // Check if multiple same ids are passed
+        // verify if passed ids are valid
+        let vm_net_ids = match &vm_config.net {
+            Some(net_configs) => net_configs.iter().map(|v| v.id.as_ref().unwrap().clone()).collect(),
```

Comment:
Please add a comment here saying that why is it okay to unwrap because you are guaranteed to have a net id in the config.

Reply:
Added.